### DISCUSSION TITLE
Table scrollbar

### DISCUSSION THREAD

**Discussion by Barugon** [emilk/egui#1743] [Q&A] [answered]
Is it possible to change the color of the scrollbar used for egui_extras::Table?

**Comment by willser** [comment]
There is a disscussion in [here  ](https://github.com/emilk/egui/discussions/1593) about `Scrollbar`'s style.

`Scrollbar`  is not customizable enough for now.But if you just want to change color,you could try following code:
````rust
ui.style_mut().visuals.widgets.active.bg_fill = Color32::BLACK;
ui.style_mut().visuals.widgets.inactive.bg_fill = Color32::BROWN;
ui.style_mut().visuals.widgets.hovered.bg_fill = Color32::BLACK;

TableBuilder::new(ui)
    .column(Size::remainder().at_least(100.0))
    .column(Size::exact(40.0))
....
````

This is not an elegant solution,I just want to show how to do it easier.

**Reply by Barugon** [reply]
> There is a disscussion in [here ](https://github.com/emilk/egui/discussions/1593) about `Scrollbar`'s style.
> 
> `Scrollbar` is not customizable enough for now.But if you just want to change color,you could try following code:
> 
> ```rust
> ui.style_mut().visuals.widgets.active.bg_fill = Color32::BLACK;
> ui.style_mut().visuals.widgets.inactive.bg_fill = Color32::BROWN;
> ui.style_mut().visuals.widgets.hovered.bg_fill = Color32::BLACK;
> 
> TableBuilder::new(ui)
>     .column(Size::remainder().at_least(100.0))
>     .column(Size::exact(40.0))
> ....
> ```
> 
> This is not an elegant solution,I just want to show how to do it easier.

Thanks for that.

**Comment by Barugon** [comment, marked as answer]
Solved it by setting `visuals.extreme_bg_color`.
